tests/conftest.py

```diff
@@ -4,9 +4,13 @@
 def pytest_addoption(parser):
     parser.addoption("--endpoint", action="append",
         help="register endpoints to test")
-
+    parser.addoption("--register_register_endpoint", help="register endpoints to test")
```

Comment:
is this the right description? I thought your help text above "URL of register register" was more accurate, but it might be useful to be specific: "URL of register register which has record for register under test" or something similar